src/main/java/org/itstep/j2_16/controller/ProductCategoryController.java

```diff
@@ -0,0 +1,30 @@
+package org.itstep.j2_16.controller;
+
+import org.itstep.j2_16.entity.ProductCategory;
+import org.itstep.j2_16.service.ProductCategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Plz, do not use wildcard import.
Plz, google how to disable wildcard import in this Intellij